boards/arm/nucleo_f103rb/nucleo_f103rb.yaml

```diff
@@ -6,3 +6,5 @@ toolchain:
   - zephyr
   - gccarmemb
 ram: 20
+supported:
+  - pwm
```

Comment:
I guess this means the board supports PWM.

Is there any way we can get this from the DTS or the pin mux information, perhaps as future work? I've  been thinking about how the "supported features" lists in the boards documentation tends to bitrot and how to fix that, and this seems like yet another place that can get out of sync with the code.

Reply:
We really should get this from DT, we need to get sanitycheck to be DT aware

Reply:
I agree and I am fine getting this from DTS once we have enough coverage of all drivers and all boards, right now this is not the case, so until then...

Reply:
> I've been thinking about how the "supported features" lists in the boards documentation tends to bitrot and how to fix that, and this seems like yet another place that can get out of sync with the code.

yeah, I was thinking about the same originally, i.e. the supported hw table in the board documentation could be generated from DTS.

Reply:
> yeah, I was thinking about the same originally, i.e. the supported hw table in the board documentation could be generated from DTS.

It should be doable with another Sphinx extension once the Python DTS parsing is in place.